docker-entrypoint.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
+set -o xtrace
 
 echo "Migrate"
-python manage.py migrate
+#python manage.py migrate
```

Comment:
Was this done on purpose?

Reply:
Unfortunately that shouldn't have been comitted - my mistake... Already "fixed" this by https://github.com/toladata/TolaActivity/commit/10c70e7526926fdb8c6e47d2e16be1658a6bf06f

Reply:
OK Thanks